src/axolotl/utils/data.py

```diff
@@ -928,8 +928,12 @@ def load_split(dataset_cfgs, _cfg):
                     )
                     split_datasets.insert(i, ds)
             else:
+                ds_type = get_ds_type(ds_cfg)
                 ds = load_dataset(  # pylint: disable=invalid-name
-                    ds_cfg["path"],
```

Comment:
I'm a little wary of completely removing a working code path. We do t have a lot of test coverage of the various file loading use cases so my confidence this won't break existing use cases is low.

Reply:
Yes. You are right.
The sft code is much more complicated.
But how do we fix this issue?
Should the user be forced to use jsonl if he or she wants to use a locale file?